diff --git a/lib/encode.c b/lib/encode.c
index 5ea29ff..7e8ff49 100644
--- a/lib/encode.c
+++ b/lib/encode.c
@@ -13,8 +13,8 @@
 #include <internal.h>
 
 #include <stdio.h>
-#include <pwd.h>
-#include <grp.h>
+// #include <pwd.h>
+// #include <grp.h>
 #include <sys/types.h>
 
 #ifdef STDC_HEADERS
@@ -157,13 +157,10 @@ th_set_device(TAR *t, dev_t device)
 void
 th_set_user(TAR *t, uid_t uid)
 {
-	struct passwd *pw;
 
-	pw = getpwuid(uid);
-	if (pw != NULL)
-		strlcpy(t->th_buf.uname, pw->pw_name, sizeof(t->th_buf.uname));
+	strlcpy(t->th_buf.uname, "0", sizeof(t->th_buf.uname));
 
-	int_to_oct(uid, t->th_buf.uid, 8);
+	int_to_oct(0, t->th_buf.uid, 8);
 }
 
 
@@ -171,13 +168,9 @@ th_set_user(TAR *t, uid_t uid)
 void
 th_set_group(TAR *t, gid_t gid)
 {
-	struct group *gr;
+	strlcpy(t->th_buf.gname, "0", sizeof(t->th_buf.gname));
 
-	gr = getgrgid(gid);
-	if (gr != NULL)
-		strlcpy(t->th_buf.gname, gr->gr_name, sizeof(t->th_buf.gname));
-
-	int_to_oct(gid, t->th_buf.gid, 8);
+	int_to_oct(0, t->th_buf.gid, 8);
 }
 
 
